### PR TITLE
Added pretty printing of json responses.

### PR DESCRIPTION
**Pull request by crantok**
Should make debugging a little easier

### CODE REVIEW

File: remote.go [hunk 1]
```diff
@@ -193,6 +193,17 @@ func (wd *remoteWD) execute(method, url string, data []byte) ([]byte, error) {
 	}
 
 	buf, err := ioutil.ReadAll(response.Body)
+
+	// Are we in debug mode, and did we read the response Body successfully?
+	if debugFlag && err == nil {
+		// Pretty print the JSON response
+		prettyBuf := bytes.NewBufferString("")
```
Comment (tebeka) [inline remote.go]: `var prettyBuf bytes.Buffer`

File: remote.go [hunk 2]
```diff
@@ -193,6 +193,17 @@ func (wd *remoteWD) execute(method, url string, data []byte) ([]byte, error) {
...
+		err = json.Indent(prettyBuf, buf, "", "    ")
+		if prettyBuf.Len() > 0 {
```
Comment (tebeka) [inline remote.go]: Also check for `err != nli`

**Comment by tebeka** [comment]
Thanks! Two minor comments and we're good to go.

**Comment by crantok** [comment]
Is that what you meant? Or should an error in pretty printing be reported to the client?

**Comment by tebeka** [comment]
Yes thanks!

**Pull request merged**